todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/source/local/TasksLocalDataSource.kt

```diff
@@ -80,22 +81,24 @@ class TasksLocalDataSource private constructor(context: Context) : TasksDataSour
         val cursor = db.query(
                 TABLE_NAME, projection, "$COLUMN_NAME_ENTRY_ID LIKE ?", arrayOf(taskId), null,
                 null, null)
-
+        var task: Task? = null
         with(cursor) {
             if (moveToFirst()) {
                 val itemId = getString(getColumnIndexOrThrow(COLUMN_NAME_ENTRY_ID))
                 val title = getString(getColumnIndexOrThrow(COLUMN_NAME_TITLE))
                 val description = getString(getColumnIndexOrThrow(COLUMN_NAME_DESCRIPTION))
-                val task = Task(title, description, itemId).apply {
+                 task = Task(title, description, itemId).apply {
                     isCompleted = getInt(getColumnIndexOrThrow(COLUMN_NAME_COMPLETED)) == 1
                 }
-                callback.onTaskLoaded(task)
-            } else {
-                callback.onDataNotAvailable()
             }
             close()
         }
         db.close()
+        if(task!=null){
+            task?.let { callback.onTaskLoaded(it) }
+        }else {
```

Comment:
Done!! 
Could you please let me know if there is any resource for format guides like newline, tabs and spaces, so that I can add those rules and achieve the format using Reformat Code menu in Android Studio.